packages/core/src/client/sitecore-client.ts

```diff
@@ -283,6 +298,21 @@ export class SitecoreClient implements BaseSitecoreClient {
           .join('/')}`;
   }
 
+  /**
+   * Execute a raw GraphQL request using the client's configured GraphQL endpoint(s).
```

Comment:
It will use only a single "Edge" endpoint
```suggestion
   * Execute a raw GraphQL request using the client's configured GraphQL Edge endpoint.
```